app/controllers/staff/schemes_controller.rb

```diff
@@ -7,6 +7,7 @@ def new
   def create
     @estate = Estate.find(estate_id)
```

Comment:
Probably not something for now, but this feels complicated enough now that it could be broken out into a helper class?

Reply:
Agree that could be a good refactor. In the interest of getting this over to users, I won't make this change just now, but I've added it to my list to look at if I get time. 👍